client/src/components/Model.js

```diff
@@ -707,4 +721,31 @@ export default class Model {
   filterClientSideFields(...additionalFields) {
     return Model.filterClientSideFields(this, ...additionalFields)
   }
+
+  static isAuthorized(user, customSensitiveInformationField) {
+    // Admins are always allowed
+    if (user?.isAdmin()) {
+      return true
+    }
+    // Else user has to be in the authorizationGroups
+    const userAuthGroupUuids =
+      user?.position?.authorizationGroups.map(ag => ag.uuid) || []
+    const fieldAuthGroupUuids =
+      customSensitiveInformationField?.authorizationGroupUuids || []
+    return fieldAuthGroupUuids.some(uuid => userAuthGroupUuids.includes(uuid))
+  }
+
+  static getAuthorizedSensitiveFields(
+    isAuthorizedCallback,
+    user,
+    customSensitiveInformation,
+    ...args
+  ) {
+    const authorizedFieldsConfig = {}
+    Object.entries(customSensitiveInformation).forEach(([k, v]) => {
```

Comment:
More descriptive names could be used for k & v.

Reply:
IMHO it is clear enough that it just creates a new object from the input by filtering some entries. So the return value is just a part of `customSensitiveInformation`.